nistats/tests/test_glm_reporter/glm_reporter_examples/second_level_models/report_oasis.py

```diff
@@ -0,0 +1,114 @@
+    # Specify and fit the second-level model when loading the data, we
+    # smooth a little bit to improve statistical behavior
+    second_level_model = SecondLevelModel(smoothing_fwhm=2.0, mask=mask_img)
+    second_level_model.fit(gray_matter_map_filenames,
+                           design_matrix=design_matrix)
+    # Estimate the contrast is very simple. We can just provide the column
+    # name of the design matrix.
+    z_map = second_level_model.compute_contrast(second_level_contrast=[1, 0, 0],
+                                                output_type='z_score')
+    # We threshold the second level contrast at uncorrected p < 0.001 and plot it.
+    # First compute the threshold.
+    _, threshold = map_threshold(
+        z_map, level=.05, height_control='fdr')
+    # Can also study the effect of sex: compute the stat, compute the
+    # threshold, plot the map
+    z_map = second_level_model.compute_contrast(second_level_contrast='sex',
+                                                output_type='z_score')
+    _, threshold = map_threshold(
+        z_map, level=.05, height_control='fdr')
+    z_maps = {'age': z_map}
+    return second_level_model, z_maps
+    
+    
+def make_design_matrix(oasis_dataset, n_subjects):
+    age = oasis_dataset.ext_vars['age'].astype(float)
+    sex = oasis_dataset.ext_vars['mf'] == b'F'
+    intercept = np.ones(n_subjects)
+    design_matrix = pd.DataFrame(np.vstack((age, sex, intercept)).T,
+                                 columns=['age', 'sex', 'intercept'])
+    design_matrix = pd.DataFrame(design_matrix, columns=['age', 'sex',
+                                                         'intercept']
+                                 )
+    return design_matrix
+
+
+def run_reporter(mask_img, design_matrix, contrast, z_maps):
+    icbm152_2009 = datasets.fetch_icbm152_2009()
+    output_filepath = make_reporter_example_result_path(__file__)
+    generate_subject_stats_report(
+            output_filepath,
+            contrasts=contrast,
+            z_maps=z_maps,
+            mask=mask_img,
+            design_matrices=[design_matrix],
+            anat=icbm152_2009['t1'],
+            )
+
+
+def get_zmap(mask_img, gray_matter_map_filenames, design_matrix, contrast):
+    zmap_filepath = os.path.join(os.path.dirname(__file__), 'oasis_zmap.nii.gz')
+    try:
+        z_map_ = nibabel.load(zmap_filepath)
+    except IOError:
+        second_level_model, z_maps = make_stats_model(mask_img,
+                                                      gray_matter_map_filenames,
+                                                      design_matrix,
+                                                      )
+        z_maps = make_zmaps(second_level_model, contrast)
+        z_maps['age'].to_filename(zmap_filepath)
+    else:
+        z_maps = {'age': z_map_}
+    return z_maps
+
+
+def create_report_oasis():
+    n_subjects = 100  # more subjects requires more memory
+    contrast = {'age': [1, 0, 0]}
```

Comment:
make that contrasts={'age':[1, 0, 0], 'sex':[0,1,0]}  to show an example with 2 contrasts